.github/test-spec.yml

```diff
@@ -2,18 +2,31 @@
 
 "CI-iot-samples-test":
   - "**/*"
+  - "!softdevice_controller/include/**/*"
+  - "!softdevice_controller/lib/**/*"
+  - "!mpsl/include/**/*"
+  - "!mpsl/lib/**/*"
 
 "CI-iot-libraries-test":
   - "**/*"
+  - "!softdevice_controller/include/**/*"
+  - "!softdevice_controller/lib/**/*"
+  - "!mpsl/include/**/*"
+  - "!mpsl/lib/**/*"
 
 "CI-lwm2m-test":
   - "**/*"
```

Comment:
These test can be skipped when SDC and mpsl is changed.

Reply:
Isn't that ensured by using the exclusion lines below?